@ndlib/gatsby-theme-marble/static/helpers.js

```diff
@@ -0,0 +1,60 @@
+import typy from 'typy'
+
```

Comment:
Is `static` a new directory? This seems like it should be in `@ndlib/gatsby-theme-marble/utils` and imported from `utils/helpers` instead for consistency.

Reply:
Good call. Moved to `utils`